css/components/input.css

```diff
@@ -0,0 +1,75 @@
+/*
+|| Input
+*/
+.input-box{
+    width: 100%;
+}
+
+.field-item{
+    width: 100%;
+    font-size: 1rem;
+    line-height: 1.5;
+    color: var(--light-black);
+    border: 1px solid var(--light-black);
+    border-radius: 0.25rem;
+    padding: 0.3rem 0.5rem;  
+}
+
+.field-item:focus{
+    border-color:var(--light-blue);
+    box-shadow:0 0 4px 0 var(--light-blue);
+}
+
+.label{
```

Comment:
Avoid giving class name same as tag name.

Reply:
as you suggest to me. i changed